Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -0,0 +1,301 @@
+  gdjs.TweenRuntimeBehavior._tweenList[owner][identifier] = new gdjs.TweenRuntimeBehavior.ObjectVariableTweenInstance(instance, false, 0, variable, currentValue);
+};
+
+gdjs.TweenRuntimeBehavior._addObjectPositionTween = (owner, identifier, instance, x, y) => {
+  gdjs.TweenRuntimeBehavior._tweenList[owner] = { identifier };
+  gdjs.TweenRuntimeBehavior._tweenList[owner][identifier] = new gdjs.TweenRuntimeBehavior.ObjectPositionTweenInstance(instance, false, 0, x, y);
+};
+
+gdjs.TweenRuntimeBehavior._getTween = (owner, identifier) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier];
+};
+
+gdjs.TweenRuntimeBehavior._tweenExists = (owner, identifier) => {
+  if (owner in gdjs.TweenRuntimeBehavior._tweenList) {
+    if (identifier in gdjs.TweenRuntimeBehavior._tweenList[owner]) {
+      return true
+    }
+  }
+  return false;
+};
+
+gdjs.TweenRuntimeBehavior._tweenIsPlaying = (owner, identifier) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].instance.isPlaying();
+};
+
+gdjs.TweenRuntimeBehavior._pauseTween = (owner, identifier) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].instance.pause();
+};
+
+gdjs.TweenRuntimeBehavior._resumeTween = (owner, identifier) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].instance.resume();
+};
+
+gdjs.TweenRuntimeBehavior._stopTween = (owner, identifier, JumpToDest) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].instance.stop(JumpToDest);
+};
+
+gdjs.TweenRuntimeBehavior._setTweenFinished = (owner, identifier, hasFinished) => {
+  gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].hasFinished = hasFinished;
+};
+
+gdjs.TweenRuntimeBehavior._tweenHasFinished = (owner, identifier) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].hasFinished;
+};
+
+gdjs.TweenRuntimeBehavior._setRemoveObjectFromScene = (owner, identifier, removeObjectFromScene) => {
+  return gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].removeObjectFromScene = removeObjectFromScene;
+};
+
+gdjs.TweenRuntimeBehavior._removeTween = (owner, identifier) => {
+  gdjs.TweenRuntimeBehavior._tweenList[owner][identifier].instance.dispose();
+  delete gdjs.TweenRuntimeBehavior._tweenList[owner][identifier];
+};
+
+/**
+ * Add an object variable tween.
+ * @param {object} owner The runtime object
+ * @param {behavior} tweenBehavior The tween behavior
+ * @param {string} identifier Unique id to idenfify the tween
+ * @param {objectvar} variable The object variable to store the tweened value
+ * @param {number} fromValue Start value
+ * @param {number} toVaue End value
+ * @param {string} easingValue Type of easing
+ * @param {number} durationValue Duration in milliseconds
+ * @param {boolean} destroyObjectWhenFinished Destroy this object when the tween ends
+ */
+gdjs.TweenRuntimeBehavior.addObjectVariableTween = function (
```

Comment:
Make this a function on the prototype as discussed earlier: `gdjs.TweenRuntimeBehavior.prototype.addObjectVariableTween`. Remove owner and tweenBehavior. Then, automatically for you:
* The `this` javascript keyword will refer to the behavior. 
* You can have the object by calling `this.owner`.